config/locales/en/views.yml

```diff
@@ -1,7 +1,16 @@
 en:
   layouts:
     application:
-      cookies_intro: GOV.UK uses cookies to make the site simpler.
+      cookies_intro_html: |
```

Comment:
Would putting this into two different entries help make things clearer? I think having HTML in these translation YAMLs isn't idiomatic.

e.g.
```yaml
cookies_intro_explanation: This service...
cookies_question: Do you accept...
```

Reply:
good shout, these changes have now been made